CsharpExtrasTest/Extensions/ArrayExtension2DTest.cs

```diff
@@ -5,11 +5,24 @@
 
 namespace CsharpExtrasTest.Extensions
 {
-    [TestFixture]
+    [TestFixture, Category("Unit")]
     public class ArrayExtension2DTest
     {
         [Test]
-        [Category("Unit")]
+        public void GIVEN_ArrayOfStrings_WHEN_MapByConcatenatingIndices_THEN_ResultIsAsExpected()
+        {
+            //Assemble
+            string[,] array = new string[,] { { "ZeroZero", "ZeroOne" }, { "OneZero", "OneOne" } };
+            Func<int, int, string, string> func = (i, j, s) => (s + i)+","+j;

```

Comment:
String interpolation might make this a bit clearer. At first it looked like you were summing the indices together.
Something like: `$"{s}{i},{j}"`